src/utils/createGenerateClassName.js

```diff
@@ -2,12 +2,7 @@
 import warning from 'warning'
 import type {Rule, generateClassName} from '../types'
 import StyleSheet from '../StyleSheet'
-import global from './global'
-
-const ns = '2f1acc6c3a606b082e5eef5e54414ffb'
-if (global[ns] == null) global[ns] = 0
-// In case we have more than one JSS version.
-const jssCounter = global[ns]++
+import moduleId from './moduleId'
```

Comment:
What's the use case for the moduleId? Isn't jssId replacing it?

Reply:
no, we need both, added a better comment about it https://github.com/cssinjs/jss/pull/648/commits/88ffe5e688a61e633a3a70ebc47257a89f15bca9